src/components/GlobalInfo.tsx

```diff
@@ -60,16 +118,19 @@ export const GlobalInfo = memo(function GlobalInfo() {
               <div className="grid auto-rows-fr grid-cols-2 gap-0">
                 <span>Analyzed</span>
                 <time className="text-right" dateTime={isoString} title={isoString}>
-                  {client ? dateTimeFormatShort(analyzerData.lastRunEpoch) : ""}
+                  {client ? dateTimeFormatShort(repodata2.lastRunInfo.time * 1000) : ""}
```

Comment:
Is there a reason for this being in seconds instead of milliseconds?

Reply:
Smaller number to store in the database. Otherwise i think we need an int64, which we could do, but :shrug: